src/zenml/integrations/kubeflow/orchestrators/kubeflow_orchestrator.py

```diff
@@ -186,13 +195,44 @@ def _upload_and_run_pipeline(
 
             # upload the pipeline to Kubeflow and start it
             client = kfp.Client()
-            result = client.create_run_from_pipeline_package(
-                pipeline_file_path,
-                arguments={},
-                run_name=run_name,
-                enable_caching=enable_cache,
-            )
-            logger.info("Started pipeline run with ID '%s'.", result.run_id)
+            if runtime_configuration.schedule:
+                try:
+                    experiment = client.get_experiment(pipeline_name)
+                    logger.info(
+                        "A recurring run has already been created with this pipeline. Creating new recurring run now.."
+                    )
+                except ValueError:
+                    experiment = client.create_experiment(pipeline_name)
+                    logger.info(
+                        f"Creating a new recurring run for pipeline '%s'.. ",
+                        pipeline_name,
+                    )
+                logger.info(
+                    "You can see all recurring runs under the '%s' experiment.'",
+                    pipeline_name,
+                )
+                result = client.create_recurring_run(
```

Comment:
Two things here:
- We might have some issues with the run-name here and how KFP generates that and passes it to our container
- We don't actually use any information of the schedule object here :P